### PR TITLE
tests: split docker tests

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

File: tests/docker_test.go [hunk 1]
```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (C) 2015 Red Hat, Inc.
```
Comment (masco-enovance) [inline tests/docker_test.go]: year?

Comment (lebauce) [reply tests/docker_test.go]: Those tests were in topology_test.go, I kept the original date

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-orientdb

File: tests/docker_test.go (hunk 1 above)
Comment (masco-enovance) [reply tests/docker_test.go]: ok, but the file creation date will be new right? is that fine? if yes no problem for me.

**Pull request merged**